HTML/html_completions.py

```diff
@@ -288,7 +288,7 @@ def entity_completion_list(self):
                     annotation=printed,
                     completion=entity,
                     completion_format=sublime.COMPLETION_FORMAT_TEXT,
-                    kind=self.KIND_ENTITY,
+                    kind=(sublime.KIND_ID_MARKUP, 'e', 'Entity'),
```

Comment:
This now causes the tuple to be created for each entity in the list instead of using a cached one. The class level attribute's reason was to avoid that.

Reply:
Right. I forgot. Thanks.